codefresh.yml

```diff
@@ -37,6 +47,7 @@ steps:
       condition:
         all:
           executeForMasterBranch: "'${{CF_BRANCH}}' == '${{MASTER_BRANCH}}'"
+          executeForTag: "'${{SEMVERSION_TAG}}' != ''"
```

Comment:
I think this will always be true as the build-harness always defines `SEMVERSION_TAG`

Reply:
Nevermind...

https://github.com/cloudposse/build-harness/blob/master/modules/git/bootstrap.Makefile#L23
https://github.com/cloudposse/build-harness/blob/master/modules/semver/Makefile#L15